tmt/steps/execute/internal.py

```diff
@@ -60,12 +57,20 @@
 echo "File $FILENAME stored to $TMT_TEST_DATA"
 """
 FILE_SUBMIT_NAME = "tmt-file-submit"
+RSTRNT_REPORT_RESULT_SCRIPT_PATHS = (
+    f"{bin_dir}rstrnt-report-result",
+    f"{bin_dir}rhts-report-result",
+    f"{bin_dir}tmt-report-result")
+REPORT_RESULT_TEMPLATE_NAME = "report_result_template"
+REPORT_RESULT_SETUP_NAME = "report_result_setup"
+REPORT_RESULT_TEARDOWN_NAME = "report_result_teardown"
+RSTRNT_REPORT_RESULT_SCRIPT = """{0}"""
+RSTRNT_REPORT_RESULT_OUTPUT = "reportResult"
 
 
 class ExecuteInternal(tmt.steps.execute.ExecutePlugin):
     """
     Use the internal tmt executor to execute tests
-
```

Comment:
Probably accidentaly removed blank line from the docstring

Reply:
Hmm, I don't know how I managed to do that.